homeassistant/components/iotty/const.py

```diff
@@ -0,0 +1,19 @@
+"""Constants for the iotty integration."""
+
+from __future__ import annotations
+
+import logging
+
+from homeassistant.const import Platform
+
+DOMAIN = "iotty"
+
+OAUTH2_AUTHORIZE = "https://auth.iotty.com/.auth/oauth2/login"
+OAUTH2_TOKEN = "https://auth.iotty.com/.auth/oauth2/token"
+OAUTH2_CLIENT_ID = "hass-iotty"
+
+IOTTYAPI_BASE = "https://homeassistant.iotty.com/"
+
+PLATFORMS: list[Platform] = [Platform.SWITCH]
+
+LOGGER = logging.getLogger(__package__)
```

Comment:
It's a bit weird to have both a shared `LOGGER` and individual `_LOGGER` in the modules, pick the pattern you prefer and go for it.

Reply:
Done. I have kept the individual `_LOGGER` just fyi